guides/UPGRADING.md

```diff
@@ -0,0 +1,46 @@
+Technology Upgrading Policy
+=======================
+
+## Overview
+
+Our approach to version upgrades is contingent on the technology in question. Details of which are outlined below, per technology and language.
+
+If you notice any upgrades that are not covered by this policy (including major versions of DBs becoming available), please
+raise them on the 20% board for triage.
+
+We don’t update dependencies in legacy apps unless there is a high or severe security vulnerability present (babbage, zebedee, train, brian,
+legacy florence).
+
+### Go
+
+- When a new Go version is released, wait for a patch release (i.e., `0.X.1`) before upgrading. For instance, if it's a choice between Go `1.15.11` and `1.16.0`, go with `1.15.11`.
+- Ensure the `go.mod` version is also bumped within applications.
+- We specifically should check that the [Go Docker image](https://hub.docker.com/_/golang) for that version is also available.
+
+### Java
+
+- Upgrade to the latest LTS version once a patch version is available. Check [AdoptOpenJDK](https://adoptopenjdk.net/index.html?variant=openjdk16&jvmVariant=hotspot) for latest versions.
+
+### Spring Boot
+
+- Upgrade to the latest minor version once a patch version is available - [versions here](https://mvnrepository.com/artifact/org.springframework.boot/spring-boot)
+
+### Node
+
+- We should use the latest `active` LTS version, waiting for a patch to the release first before adopting. Details can be found on the [nodejs](https://nodejs.org/en/) website.
+
+### React
+
+- Upgrade to the latest minor version once a patch is available; you can find them [here](https://reactjs.org/versions).
+
+### Frameworks and libraries
+
+- Go with the latest minor version with a patch. It is also down to senior developers to determine when major version upgrades are worth doing.
```

Comment:
Choose or upgrade, rather than Go to avoid confusion with `Golang` 😁 
```suggestion
- Choose the latest minor version with a patch. It is also down to senior developers to determine when major version upgrades are worth doing.
```

Reply:
Updated